tests/TestCase/Http/ResponseTest.php

```diff
@@ -325,9 +325,9 @@ public function testWithCache()
         $this->assertFalse($response->hasHeader('Date'));
         $this->assertFalse($response->hasHeader('Last-Modified'));
 
-        $this->assertSame(gmdate('D, j M Y G:i:s ', $since) . 'GMT', $new->getHeaderLine('Date'));
```

Comment:
I think we should just rely on the constant in test cases as well.

If we wanted to test a specifically formatted date, we should start with the raw date string, convert to time and then compare that.

Reply:
Done